README.md

```diff
@@ -1,160 +1,137 @@
-- `src`: Contains all the business logic files and the style sheets
-  - `background`: The business logic for the extension background process
-      - `listeners`: Contains all the listeners which listen on all the events happened in the browser
-          - `tabListener.ts`: The listeners which listen on all the tab related events [API](https://developer.chrome.com/docs/extensions/reference/tabs/)
-          - `webRequestListener.ts`: The listeners which listen on all the web request related events [API](https://developer.chrome.com/docs/extensions/reference/webRequest/)
-      - `providers`: Contains the provider-specific code of all the Privacy Pass providers in the extension. Currently we have only Cloudflare and hCaptcha
-      - `voprf.js`: Legacy crypto code which is still in Vanilla JavaScript
-      - `voprf.d.ts`: TypeScript declaration file for the legacy crypto code
-      - `tab.ts`: Tab class to represent a tab and encapsulate everything which is Tab specific
-      - `token.ts`: Token class to represent a token and contain all the code related to tokens
-  - `popup`: The web app for the popup in the browser toolbar
-      - `components`: Contains all the React components
-      - `styles`: Contains all the style sheets which are shared among the React components
-      - `types.d.ts`: Global Typescript declaration
-
-## Cryptography
-
-Cryptography is implemented using the elliptic-curve library
-[SJCL](https://github.com/bitwiseshiftleft/sjcl) and compression of
-points is done in accordance with the standard SEC1. This work uses the
-NIST standard P256 elliptic curve for performing operations. Third-party
-implementers should note that the outputs of the hash-to-curve, key
-derivation, and point encoding functions must match their Go equivalents
-exactly for interaction with our server implementation. More information
-about this will be provided when the edge implementation is
-open-sourced.
-
-## Acknowledgements
-
-The creation of the Privacy Pass protocol was a joint effort by the team
-made up of George Tankersley, Ian Goldberg, Nick Sullivan, Filippo
-Valsorda and Alex Davidson.
-
-We would also like to thank Eric Tsai for creating the logo and
-extension design, Dan Boneh for helping us develop key parts of the
-protocol, as well as Peter Wu and Blake Loring for their helpful code
-reviews. We would also like to acknowledge Sharon Goldberg, Christopher
-Wood, Peter Eckersley, Brian Warner, Zaki Manian, Tony Arcieri, Prateek
-Mittal, Zhuotao Liu, Isis Lovecruft, Henry de Valence, Mike Perry,
-Trevor Perrin, Zi Lin, Justin Paine, Marek Majkowski, Eoin Brady, Aaran
-McGuire, and many others who were involved in one way or another and
-whose efforts are appreciated.
+### Manually Loading Extension
+
+#### Firefox
+
+1. Open Firefox and navigate to [about:debugging#/runtime/this-firefox/](about:debugging#/runtime/this-firefox/)
+1. Click on 'Load Temporary Add-on' button.
```

Comment:
nits: why not use
```md
1.
2.
3.
4.
```

Reply:
copy-paste? but it also works. I think it is supported as it makes easier to update when changes are frequent.